### PR TITLE
Brought stores to a common view,  Amazon iap_lib update

### PR DESCRIPTION
**Pull request by SalavatR**
Brought stores to a common view
Amazon iap_lib update

### CODE REVIEW

File: extension-iap/manifests/android/AndroidManifest.xml [hunk 1]
```diff
@@ -4,9 +4,9 @@
 <uses-sdk android:minSdkVersion="{{android.minimum_sdk_version}}" android:targetSdkVersion="{{android.target_sdk_version}}" />
     <application>
         <!-- For Amazon IAP -->
-        <receiver android:name="com.amazon.device.iap.ResponseReceiver" >
+        <receiver android:name = "com.amazon.device.iap.ResponseReceiver" android:permission = "com.amazon.inapp.purchasing.Permission.NOTIFY" >
```
Comment (britzl) [inline extension-iap/manifests/android/AndroidManifest.xml:7]: Looks good. Source: https://developer.amazon.com/docs/in-app-purchasing/iap-implement-iap.html#responsereceiver

File: extension-iap/src/java/com/defold/iap/IapAmazon.java [hunk 2]
```diff
@@ -173,9 +174,13 @@ public void onProductDataResponse(ProductDataResponse productDataResponse) {
...
             listener.onProductsResult(IapJNI.BILLING_RESPONSE_RESULT_ERROR, null, commadPtr);
         } else {
+            for (final String s : productDataResponse.getUnavailableSkus()) {
+                Log.v(TAG, "Unavailable SKU: " + s);
+            }
+
             Map<String, Product> products = productDataResponse.getProductData();
             try {
-                JSONObject data = new JSONObject();
+                JSONArray data = new JSONArray();
```
Comment (britzl) [inline extension-iap/src/java/com/defold/iap/IapAmazon.java:183]: Same as: https://github.com/defold/extension-iap/blob/master/extension-iap/src/java/com/defold/iap/IapGooglePlay.java#L401-L406

**Review by britzl — APPROVED** [review]
Thank you for the contribution!

**Comment by britzl** [comment]
@SalavatR could you please sign the CLA so that we can merge your contribution?

https://cla-assistant.io/defold/extension-iap?pullRequest=42

**Comment by SalavatR** [comment]
@britzl Done)

**Pull request merged**